ProjectToTest/User.cs

```diff
@@ -1,21 +1,31 @@
-﻿using System.Reflection;
-
-namespace ProjectToTest
+﻿namespace ProjectToTest
 {
+    internal enum UserGender
```

Comment:
It is better to extract into a separate file

Reply:
It is a common practice to store different data in the different files, but this project is small and I don’t see a need in that.